### PR TITLE
ref #221: Hide the new button if it makes no sense

### PR DESCRIPTION
**Pull request by UlrichKu**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed issues  | chameleon-system/chameleon-system#221
| License       | MIT

Do not show "new button" for this field type.

### CODE REVIEW

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php [hunk 1]
```diff
@@ -119,6 +119,11 @@ public function GetHTML()
         return $html;
     }
 
+    protected function canShowNewRecordButton(): bool
```
[the remark this answers is not in the document]
Comment (UlrichKu) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php]: Ok.

Also done for the "edit check" below.

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php [hunk 2]
```diff
@@ -119,6 +118,28 @@ public function GetHTML()
         return $html;
     }
 
+    protected function isRecordCreationAllowed(string $foreignTableName): bool
+    {
+        $activeUser = &TCMSUser::GetActiveUser();
```
Comment (umulmrum) [inline src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php]: Assignment by reference was only required in PHP 4 - as this is new code, we should use the opportunity to remove the "&" (also in the method below).

Comment (UlrichKu) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php]: ok

**Comment by umulmrum** [comment]
Thanks @UlrichKu

**Pull request merged**